src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -147,28 +147,24 @@ public void RefreshWorkloadManifests()
                 }
             }
 
+            _installStateFilePath = Path.Combine(WorkloadInstallType.GetInstallStateFolder(_sdkVersionBand, _sdkRootPath), "default.json");
+            var installState = InstallStateContents.FromPath(_installStateFilePath);
```

Comment:
what happens if the file doesn't exist or is empty/malformed?

Reply:
InstallStateContents has a File.Exists check. It being malformed isn't supported, especially as users are not expected to modify their install states directly, so that's an error.

Reply:
Note that this doesn't change based on an extraneous File.Exists check, so if you want a cleaner error message for the user for malformed install state files, or perhaps some kind of fallback (empty install state?), I can implement that, but it doesn't already exist in 400, so without customer need, I don't think that meets the bar for servicing.